src/DynamoManipulation/NodeManipulator.cs

```diff
@@ -128,7 +156,7 @@ protected virtual bool CanMoveGizmo(IGizmo gizmo)
         {
             //Wait until node has been evaluated and has got new origin
             //as expected position.
-            return gizmo != null && newPosition.DistanceTo(gizmo.Origin) < 0.01;
+            return gizmo != null && newPosition.DistanceTo(Origin) < 0.01;
```

Comment:
Since the gizmo origin is now different from the point location, we need to keep in mind that all hit test calculations need to be done wrt the point location and not the gizmo location.